pkgs/desktops/gnome/apps/gnome-text-editor/default.nix

```diff
@@ -0,0 +1,76 @@
+{lib
```

Comment:
Will be nice to format the file with [`nixpkgs-fmt`](https://github.com/nix-community/nixpkgs-fmt).